docker/build/usr/bin/cc

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+
+ARTIFACTS="/usr/bin" \
+exec /lib/symbolica/cc "$@"
```

Comment:
I decided to re-introduce a wrapper script for `cc` which just set `ARTIFACTS` to `/usr/bin` as we know that's where the bits are on the docker image. That way we can keep the `cc` in `/lib/build` more flexible for those other builds.